front/app/services/homepageSettings.ts

```diff
@@ -7,6 +7,7 @@ const homepageSettingsEndpoint = `${API_PATH}/home_page`;
 // setting in appConfiguration.ts
 export type TAppConfigSectionSetting = Extract<
   THomepageEnabledSetting,
+  // still move to module
```

Comment:
I made a small change but I think it's fine to have these strings as keys here, rather than the modules. It's just type checking, not actual module code. I believe that in the dev environment (the only place TS checks types) we'll always have access to types extended in the modules, but even if not, extracting a key that doesn't exist (e.g. it came from a module that isn't installed) doesn't fail in TS so